packages/shared/src/components/squads/SelectArticle.tsx

```diff
@@ -27,7 +29,7 @@ export function SquadSelectArticle({
     key,
     query: READING_HISTORY_QUERY,
   };
-  const { data, isInitialLoading, isLoading } =
+  const { hasData, data, isInitialLoading, isLoading } =
```

Comment:
Now that I see this again, we might want to memoize it

Reply:
It's already memoized in the hook, not sure if that solves your concern?

Reply:
Ohh, is it? Then we are good to go 🥳